Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -221,6 +229,12 @@ import BraintreeDataCollector
         performSwitchRequest(appSwitchURL: url, paymentType: paymentType, completion: completion)
     }
 
+    // MARK: - App Switch Methods
+
+    func handleOpen(_ url: URL) {
```

Comment:
I know this is the naming we use in Venmo, but it's a little confusing to me since without looking through the code callsites you don't know if this method is for opening the Venmo app, or opening the merchant app.

_Take it or leave it comment_ - but we could rename to `handleReturnURL()` or something similar

Reply:
That makes sense to me too: https://github.com/braintree/braintree_ios/pull/1218/commits/2d53f54c627d7e39fdc9c9835fc8bc43029577c8